### ISSUE TITLE
Pktvisor: implement support for updating an existing policy in the pktvisor backend in orb-agent

### ISSUE THREAD

**Issue by manrodrigues** [ns1labs/orb#721]
Log level "warn" with error message when an edition is applied to a policy. Note that, despite this, the policy is modified in the backend. The reason for that is there is no implemented support for updating an existing policy in the pktvisor backend in orb-agent.

**Comment by manrodrigues** [comment]
Editing is not consistent: I was successful on the first attempt, but on the second, I got a timeout.

Steps:
-Create a policy without filter
```
            "filter": {
                "bpf": "udp port 53"
            },
```
- Edit policy inserting this filter
- Edit the policy again removing this filter

After second edition:

![image](https://user-images.githubusercontent.com/78241475/151809389-eb3defec-35f9-4214-b82d-f7fecad82367.png)

```
{"level":"info","ts":1643638317.5842543,"caller":"pktvisor/pktvisor.go:247","msg":"pktvisor stdout","log":"[2022-01-31 14:11:57.584] [visor] [info] policy [policy1]: stopping input instance: default_pcap-9ea395abea1cbc2d"}
{"level":"warn","ts":1643638327.583439,"caller":"pktvisor/pktvisor.go:155","msg":"policy failed to remove","policy_id":"15b1c401-c957-4457-9e3d-227c0c861a5d","policy_name":"policy1","error":"Delete \"http://localhost:10859/api/v1/policies/policy1\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)"}
{"level":"info","ts":1643638327.5846417,"caller":"pktvisor/pktvisor.go:247","msg":"pktvisor stdout","log":"[2022-01-31 14:12:07.584] [visor] [info] policy [policy1]: parsing"}
{"level":"warn","ts":1643638332.5840094,"caller":"pktvisor/pktvisor.go:179","msg":"yaml policy application failure","policy_id":"15b1c401-c957-4457-9e3d-227c0c861a5d","policy":"version: \"1.0\"\nvisor:\n    policies:\n        policy1:\n            handlers:\n                modules:\n                    handler_dhcp_1:\n                        type: dhcp\n            input:\n                input_type: pcap\n                tap: default_pcap\n            kind: collection\n"}
{"level":"warn","ts":1643638332.5841122,"caller":"policyMgr/manager.go:168","msg":"policy failed to apply","policy_id":"15b1c401-c957-4457-9e3d-227c0c861a5d","policy_name":"policy1","error":"Post \"http://localhost:10859/api/v1/policies\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)"}
{"level":"error","ts":1643638351.6565003,"caller":"pktvisor/pktvisor.go:297","msg":"scrape failed","error":"Get \"http://localhost:10859/api/v1/policies/__all/metrics/bucket/1\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)","stacktrace":"github.com/ns1labs/orb/agent/backend/pktvisor.(*pktvisorBackend).scrapeDefault.func1\n\t/go/src/github.com/ns1labs/orb/agent/backend/pktvisor/pktvisor.go:297\nreflect.Value.call\n\t/usr/local/go/src/reflect/value.go:556\nreflect.Value.Call\n\t/usr/local/go/src/reflect/value.go:339\ngithub.com/go-co-op/gocron.callJobFuncWithParams\n\t/go/pkg/mod/github.com/go-co-op/gocron@v1.9.0/gocron.go:78\ngithub.com/go-co-op/gocron.(*executor).start.func1.1\n\t/go/pkg/mod/github.com/go-co-op/gocron@v1.9.0/executor.go:90\ngolang.org/x/sync/singleflight.(*Group).doCall.func2\n\t/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/singleflight/singleflight.go:193\ngolang.org/x/sync/singleflight.(*Group).doCall\n\t/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/singleflight/singleflight.go:195\ngolang.org/x/sync/singleflight.(*Group).Do\n\t/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/singleflight/singleflight.go:108\ngithub.com/go-co-op/gocron.(*executor).start.func1\n\t/go/pkg/mod/github.com/go-co-op/gocron@v1.9.0/executor.go:82"}
{"level":"info","ts":1643638355.432529,"caller":"pktvisor/pktvisor.go:247","msg":"pktvisor stdout","log":"[2022-01-31 14:12:35.432] [visor] [info] policy [policy1]: stopping handler instance: policy1-handler_dhcp_1"}


```

Full logs:
[logs_edit_failed.txt](https://github.com/ns1labs/orb/files/7971398/logs_edit_failed.txt)

**Comment by manrodrigues** [comment]
@leoparente

**Comment by manrodrigues** [comment]
As the edit is working and the timeout causing the error is being analyzed and resolved in #725 , I am closing this issue
